sharepoint/sharepoint-ps/sharepoint-online/Get-SPOApplication.md

```diff
@@ -91,13 +93,13 @@ Accept wildcard characters: False
 
 ### -ApplicationId
 
-Use this parameter to enumerate permissions of the owning applications registered in the specified tenant.
+Use this parameter to enumerate app-only permissions of the guest application id with access to the specified owning application.
 
 ```yaml
 Type: String
 Parameter Sets: ParamSet2
 Aliases:
-Applicable: SharePoint Online
+Applicable: SharePoint
```

Comment:
>SharePoint [](http://example.com/codeflow?start=12&length=10)

should be "SharePoint Online"

Reply:
Got it will change, also, Are all references of SharePoint Online deprecated? For example, this is still referred to as sharepoint online management shell https://learn.microsoft.com/en-us/powershell/sharepoint/sharepoint-online/connect-sharepoint-online

Reply:
I made this change to the other yaml header as well

Reply:
@cindylay I mean it should read "SharePoint Online", not "SharePoint" -- I think in a previous iteration you removed the "Online" from here